README.md

```diff
@@ -1,2 +1,85 @@
-# scf-minibroker
-Minibroker tooling to work with SUSE Cloud Foundry
+# Assessing the minibroker database charts.
+
+The main purpose of the tooling in this repository is to assess which
+of the publicly available charts for the databases supported by
+minibroker will work with SCF.
+
+## How to use the tooling
+
+The main entry point / script is `assessor/bin/assess-minibroker-charts.rb`.
+
+The script requires:
+
+  - A working `kubectl` command in the `PATH`, with its config
+    pointing to
+  - A working SCF+UAA cluster
+  - A working `cf` client command in the `PATH`. The tooling will
+    target it to the CAP cluster.
+  - A working `patch` command in the `PATH`. For a vagrant box this
+    means that is is necessary to run `sudo zypper install patch`
+    before attempting an assessment.
+  - A working `ruby`, of course, and the gems
+      - fileutils
+      - net/http
+      - open3
+      - optparse
+      - uri
+      - yaml
```

Comment:
Actually those are all part of the standard library, so we don't need to install them separately. (Unless you're crazy enough to be trying to use [mruby](http://mruby.org/), but I don't think anybody would expect anything to ever work in that case.)

Reply:
ok.

Reply:
Adressed with commit b671d3d1eddb106ceee500f0c59ad31eb3b0cba1